### PR TITLE
Linking to translations, chapters and book possible

### PR DESCRIPTION
**Pull request by SantiagoDePolonia**

### CODE REVIEW

**Comment by clash82** [comment]
Hey @SantiagoDePolonia this looks great, thank you.

Unfortunately, it's not working when you open page with already predefined path (eg. `/b/pl_bt3/joe/3`), it always redirects you to the default location (`/b/pl_pubg/joh/1`). Can we do something with this here or is it something for later improvement?

**Comment by SantiagoDePolonia** [comment]
Right!
I didn't test this scenario.
I added a fix for that.

File: assets/js/Bible.js [hunk 1]
```diff
@@ -3,6 +3,11 @@ import Navigator from "./Navigator";
 import Reader from "./Reader";
 import StatusBar from "./StatusBar";
 
+const URL_PREFIX = '/b';
+const DEFAULT_TRANSLATION = 'pl_pubg';
+const DEFAULT_BOOK = 'joh';
+const DEFAULT_CHAPTER = '1';
```
Comment (clash82) [inline assets/js/Bible.js:9]: With those constants everything is now more clear +1

**Review by clash82 — APPROVED** [review]
Good work Kuba! I like that you enriched this code with your comments.

**Comment by clash82** [comment]
@SantiagoDePolonia No i mamy aktualizację adresów URL. @SantiagoDePolonia ❤️

**Pull request merged**